src/main/java/no/unit/bibs/contents/UpdateContentsApiHandler.java

```diff
@@ -13,21 +13,40 @@
 import org.apache.http.HttpStatus;
 import org.slf4j.LoggerFactory;
 
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.MalformedURLException;
+import java.net.URL;
+
 import static java.util.Objects.isNull;
 
 public class UpdateContentsApiHandler extends ApiGatewayHandler<ContentsRequest, GatewayResponse> {
 
+    public static final String OBJECT_KEY_TEMPLATE = "/%s/%s/%s/%s";
```

Comment:
Oh man, du elsker String.format(). But why not!
String.join(..., ..., ...., "/") gjør det samme ;)